### PR TITLE
CZML Support

### PR DESCRIPTION
**Pull request by joschne**
This PR aims to add CZML Support.

**Why** 
CZML provides an easy way to create time dynamic entities (see [CZML Guide](https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/CZML-Guide) and [Cesium.CzmlDataSource](https://cesiumjs.org/Cesium/Build/Documentation/CzmlDataSource.html?classFilter=Czm) or this [example](https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=CZML%20Point%20-%20Time%20Dynamic.html&label=CZML)).

Currently it is not possible to use CZML / Cesium.CzmlDataSource with ac-layer (see #231). 

**How** 
This PR adds
* New `<ac-czml-desc>` component that accepts a stream of CZML packets.
* New `czml-drawer` service that processes and draws CZML packets to CesiumViewer.
* New `<czml-layer>` component in demo app that shows usage of `<ac-czml-desc>`.

**Remarks** 

To see a working example, use the demo app and do this:
- Uncomment `<czml-layer></czml-layer>` in `demo-map.component.html`
- In `viewerOptions` of `demo-map.component.ts` enable `timeline`, `animation` and `shouldAnimate` by setting their values true

`czml-drawer` service does not use OptimizedEntityCollection (to keep things simple). I don't know the implications, please pay special attention to this aspect when reviewing the PR.

### CODE REVIEW

File: src/angular-cesium/components/ac-layer/ac-layer.component.ts [hunk 1]
```diff
@@ -272,33 +276,33 @@ export class AcLayerComponent implements OnInit, OnChanges, AfterContentInit, On
...
 		this.layerService.options = this.options;
 		this.layerService.show = this.show;
 		this.layerService.zIndex = this.zIndex;
 		this._drawerList.forEach((drawer, drawerName) => {
 			const initOptions = this.options ? this.options[drawerName] : undefined;
 			const drawerDataSources = drawer.init(initOptions);
 			// only entities drawers create data sources
-      // TODO: Causes Bad Performance
-      // if (drawerDataSources) {
+			if (drawerDataSources) {
```
Comment (eitanfr) [inline src/angular-cesium/components/ac-layer/ac-layer.component.ts]: @tomermoshe why did you put this code in comment? how does it cause bad performance and is it ok to uncomment it?

**Comment by eitanfr** [comment]
can you please fix CI tests?

**Comment by joschne** [comment]
I did some simple change in order to rerun CI test. 
Now it passed CI successfully.

File: demo/app/components/demo-map/demo-map.component.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ export class DemoMapComponent {
 			fullscreenButton : false,
 			baseLayerPicker : false,
 			animation : false,
+			shouldAnimate: false,
```
Comment (tomermoshe) [inline demo/app/components/demo-map/demo-map.component.ts]: according to cesium docs the default value of viewer.shouldAnimate is false: https://cesiumjs.org/Cesium/Build/Documentation/Viewer.html

**Comment by tomermoshe** [comment]
Please resolve conflicts

**Pull request merged**